main.go

```diff
@@ -123,6 +42,11 @@ func main() {
 		logLevel      = flag.String("log-level", "info", "One of trace, debug, info, warn, error, fatal, or panic.")
 	)
 
+	tracerCtx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	shutdown := otelutils.TracerProviderFromEnv(tracerCtx, serviceName, func(e error) { log.Fatal(e) })
```

Comment:
Since this calls out to `log.Fatal` it's probably ideal to put it after `logging.SetupLogging`. It's not that likely to matter, but again, cleaner. It's fine to do it there, because `flag.Parse` and `logging.SetupLogging` aren't going to have any spans involved in them, heh.

Reply:
Thanks, should be fixed now.